omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js

```diff
@@ -602,11 +602,13 @@ ConfirmationSectionModel.prototype.select = function() {
     })
 
     if (!hasData) {
-        this.questions[0].confirm.disable();
```

Comment:
@vishnuraom the code you are replacing looks more generic than yours. Have you tried to investigate what was wrong with it? Could the problem be somewhere else instead?